src/Appwrite/Platform/Workers/StatsResources.php

```diff
@@ -129,9 +129,20 @@ protected function countForProject(Database $dbForPlatform, callable $getLogsDB,
             ]);
             $teams = $dbForProject->count('teams');
             $functions = $dbForProject->count('functions');
+
             $messages = $dbForProject->count('messages');
             $providers = $dbForProject->count('providers');
             $topics = $dbForProject->count('topics');
+            $targets = $dbForProject->count('targets');
+            $emailTargets = $dbForProject->count('targets', [
+                Query::equal('providerType', [MESSAGE_TYPE_EMAIL])
+            ]);
+            $pushTargets = $dbForProject->count('targets', [
+                Query::equal('providerType', [MESSAGE_TYPE_PUSH])
+            ]);
+            $smsTargets = $dbForProject->count('targets', [
+                Query::equal('providerType', [MESSAGE_TYPE_SMS])
+            ]);
```

Comment:
Would be much quicker (and potentially more consistent) to use array_filter on the PHP side than to re-count for each type

Reply:
@abnegate I'm not sure what you mean? We need to count for each type, get count by type, do we support groupBy? If not how will it work?